packages/console/src/cloud/pages/Main/TenantLandingPage/TenantLandingPageContent/index.tsx

```diff
@@ -52,7 +53,7 @@ function TenantLandingPageContent({ className }: Props) {
         />
       </div>
       <CreateTenantModal
-        skipPlanSelection
+        skipPlanSelection={!isDevFeaturesEnabled}
```

Comment:
It seems we are going to deprecate the skipPlanSection prop itself.

Reply:
Yes, this will be removed when the whole feature is ready.